packages/w3up-client/src/client.js

```diff
@@ -233,12 +235,39 @@ export class Client extends Base {
 
   /**
    * Create a new space with a given name.
-   *
+   * If an account is provided in the options argument, then it creates a delegated recovery account.
+   * 
+   * @typedef {object} CreateOptions
+   * @property {string|false} [account]
+   * 
    * @param {string} name
+   * @param {CreateOptions} options
    */
-  async createSpace(name) {
-    return await this._agent.createSpace(name)
+  async createSpace(name, options = {}) {
```

Comment:
at this API level, I think this should take an actual account object, rather than an email address. 

the code to obtain the account from the email can be left as an exercise above this code level.

also, the default flow for setting up the client often contains and instruction to login:

```
const myAccount = await client.login('zaphod@beeblebrox.galaxy')
```

So often the JS user already has the account.